compiler-core/src/language_server/code_action.rs

```diff
@@ -411,3 +411,298 @@ impl<'a> LetAssertToCase<'a> {
         self.actions
     }
 }
+
+/// Code action to desugar "use" expressions.
+///
+/// Turns:
+/// ```gleam
+/// use x <- result.map(Ok(1))
+/// io.debug(x)
+/// ```
+///
+/// Into:
+/// ```gleam
+/// result.map(Ok(1), fn(x) { io.debug(x) })
+/// ```
+///
+/// The code action is available for the i'th subject if:
+/// - the whole "use" expression body overlaps with the selection.
+/// - if multiple "use" expressions match, only the inner-most is updated.
```

Comment:
This doesn't seem to be a condition for the code action being available, yet it is in the list of conditions.